host/4/dotnet-build.yml

```diff
@@ -128,6 +128,48 @@ steps:
     displayName: dotnet-isolated-appservice (.NET 6)
     continueOnError: false
 
+  - bash: |
```

Comment:
FYI: I do not think we need [a core-tools version file](https://github.com/Azure/azure-functions-docker/blob/dev/host/4/bullseye/amd64/dotnet/dotnet-isolated/dotnet-isolated-core-tools.Dockerfile) with .NET7 changes, hence did not include that in this PR. Please correct me If I am wrong.